packages/amplication-server/src/core/git/git.provider.service.ts

```diff
@@ -57,7 +60,12 @@ export class GitProviderService {
       limit: args.limit,
       page: args.page,
     };
-    const gitProviderArgs = { provider: args.gitProvider, installationId };
+    const gitProviderArgs = {
+      provider: args.gitProvider,
+      providerProperties: {
+        installationId,
+      },
+    };
```

Comment:
It will be better to have a single function to get the GitClientService so all functions in this files can use a single method to get the client